doc/bu-electrum-integration.md

```diff
@@ -1,31 +1,31 @@
-Electrum Server Basic integration.
-==================================
+# Electrum Server
 
-Starting from version 1.6  BUcash it's possible to build a rust electrum server implementation binary (electrs)
-and spawn the process during the startup phase. Then you could use this electum server
-along with the electron cash wallet.
+## Overview
 
-To build the [electrs](https://github.com/dagurval/electrs) binaries use:
+Bitcoin Unlimited includes support for electrum server (from version 1.6, Linux x64 only).
 
-```sh
-make electrs
-```
+The server is maintained as a [separate piece of software](https://github.com/bitcoinunlimited/electrscash), but is integrated with the node software. Its process has the same lifetime as the bitcoind process, uses the same log files and adds functionality to the same RPC interface.
 
-To make it so that `bitcoind` spawn `electrs` at startup you need to add those
-lines to your `bitcoin.conf`:
+Running a private electrum server gives you extend API interface, for example allowing you to lookup transaction history and balances of any address on the blockchain. Having a private electrum server also allows you to use wallet software such as Electron Cash, without compromising your privacy.
 
-```
-electrum=1 # to start the electrum server
-electrum.exec=path/to/bin/electrs # optional path to your electrum server binary
-debug=electrum
-```
+By [enabling SSL support](https://github.com/BitcoinUnlimited/ElectrsCash/blob/master/doc/usage.md) and making the server public, you're able to support the network light client infrastructure, which is run on volunteer basis (you may want to [add it to the server list](https://github.com/Electron-Cash/Electron-Cash/blob/master/lib/servers.json) in that case)
+
+## Starting server
 
-Adding `debug=electrum` will enable more useful logging.
+Add `electrum=1` to your bitcoin.conf file, or pass `-electrum=1` argument to bitcoind when from command line. You may want to also add `debug=electrum` to enable useful logging.
 
-This is how a local electrum cash wallet could be connected to the electrum server:
+The RPC call `getelectruminfo` gives you runtime details, such as indexing status of the server. More detailed metrics are available via Prometheus, see monitoring section of the [usage document](https://github.com/BitcoinUnlimited/ElectrsCash/blob/master/doc/usage.md)
+
+## Compiling from source
+
+The latest stable release used by the node software can be compiled from the Bitcoin Unlimited source tree by running
 
 ```sh
-./electron-cash -v --oneserver --server=127.0.0.1:60001:t --testnet
+make electrs
 ```
 
-Known issue: your RPC password is shown in the electrs command line.
+For more recent development versions, see the [usage document](https://github.com/BitcoinUnlimited/ElectrsCash/blob/master/doc/usage.md).
```

Comment:
I read the content of that usage document and I think it need to be updated, cause it makes reference to Bitcoin Core as a prereq and - It makes reference to Electrum wallet rather than Electron Cash .

Reply:
sorry for  the noise, @dagurval just point me to an actual [PR](https://github.com/BitcoinUnlimited/ElectrsCash/pull/10/files) that is taking care of it on the BitcoinUnlimited/ElectrsCash repo